test/indexer/test_tdr.py

```diff
@@ -180,6 +180,23 @@ def test_fetch_bundle(self):
         with self.assertRaises(RequirementError):
             self._test_fetch_bundle(bundle, load_tables=False)
 
+    def test_disallow_double_pipe_joiner(self):
+        bundle = self._load_canned_bundle(self.bundle_fqid)
+        self._test_fetch_bundle(bundle, load_tables=True)
```

Comment:
We just need to load the tables, not repeat the `fetch_bundle` test
```suggestion
        self._make_mock_tdr_tables(bundle.fqid)
```